package.json

```diff
@@ -39,17 +42,19 @@
     "react-dom": "18.3.1",
     "react-native": "0.76.6",
     "react-native-gesture-handler": "~2.20.2",
-    "react-native-reanimated": "~3.16.1",
+    "react-native-reanimated": "^3.16.7",
     "react-native-safe-area-context": "4.12.0",
-    "react-native-screens": "~4.4.0",
+    "react-native-screens": "^4.6.0",
     "react-native-web": "~0.19.13",
     "react-native-webview": "13.12.5"
   },
   "devDependencies": {
     "@babel/core": "^7.25.2",
+    "@react-native-community/cli": "latest",
```

Comment:
Unneeded package https://github.com/react-native-community/cli/tree/main/packages/cli
"This package contains source code for @react-native-community/cli, the actual CLI that comes bundled with React Native. You don't need to install it separately in your project."